lib/modules/dosomething/dosomething_campaign/includes/Campaign.php

```diff
@@ -51,6 +51,7 @@ public static function get($ids, $display = 'teaser') {
     $results = node_load_multiple($ids);
 
     if (!$results) {
+      http_response_code(404);
       throw new Exception('No campaign data found.');
```

Comment:
I'd imagine we want to keep consistent with an `error` object with a `message`, so it looks like the `throw new Exception` does that:  http://staging.beta.dosomething.org/api/v1/signups/abc 

cc Sir RESTful @DFurnes